### PR TITLE
Remove magic number and add const maxRelayFeeMultiplier

### PR DESCRIPTION
**Pull request by alexlyp**

### CODE REVIEW

File: mempool.go [hunk 1]
```diff
@@ -93,6 +93,10 @@ const (
 	// required for a transaction to enter the mempool on TestNet or SimNet.
 	minTxFeeForMempoolTestNet = 1e2
 
+	// maxRelayFeeMultiplier is the factor that we disallow fees / kb above
+	// the minimum tx fee.
+	maxRelayFeeMultiplier = 100
```
Comment (raedah) [inline mempool.go]: Looks like this would block https://github.com/decred/dcrwallet/pull/214

**Comment by alexlyp** [comment]
How so @raedah?  It is just replacing the magic number of 100

**Comment by raedah** [comment]
Seeing that this is in mempool.go, and the description is "maxRelayFeeMultiplier is the factor that we disallow fees / kb above the minimum tx fee", it sounds like transactions with large fees will be blocked from being relayed or put into the mempool. So if I wanted to do a large fee transaction, and used the --allowhighfees flag, the rest of the network would drop my transaction instead of propagating it.

**Comment by alexlyp** [comment]
The only way allowhighfees can be set to true is with the sendrawtransaction rpc.  So this only protects your connected wallet, but does nothing from propogating other peoples' high fee tx.

**Comment by cjepson** [comment]
utACK, OK for merge

**Pull request merged**